.changeset/stinky-shoes-melt.md

```diff
@@ -0,0 +1,15 @@
+---
+"fluid-framework": minor
+---
+---
+"section": "tree"
+---
+Export alpha index APIs
+
+This exposes two new index APIs, `createSimpleTreeIndex` and `createIdentifierIndex`.
+
+`createSimpleTreeIndex` can be used to create a `SimpleTreeIndex` that indexes nodes based on their schema.
+Depending on the schema, the user specifies which field to key the node on.
```

Comment:
```suggestion
The developer can specify the schema field on which to key the index.
```

Reply:
this suggestion is a little inaccurate because the user must specify the field for a particular schema or nodes of that schema won't be included in the index

maybe something more like:
```suggestion
To index nodes of a particular schema, the developer must specify the schema field which to key the nodes.
```